src/main/java/uk/gov/companieshouse/company_appointments/CompanyAppointmentMapper.java

```diff
@@ -27,11 +27,13 @@
 
     public CompanyAppointmentView map(CompanyAppointmentData companyAppointmentData) {
         LOGGER.debug("Mapping data for appointment: " + companyAppointmentData.getId());
+        boolean isSecretary = "secretary".equals(companyAppointmentData.getData().getOfficerRole());
```

Comment:
A secretary can be any of the following:

* secretary
* corporate-secretary
* nominee-secretary
* corporate-nominee-secretary